src/_gettsim/parameters/unterhalt.yaml

```
@@ -15,73 +15,61 @@ mindestunterhalt:
     12: 393
-    17: 460
+    18: 460
     reference: V. v. 03.12.2015 BGBl. I S. 2188.
   2018-01-01:
     6: 348
     12: 399
-    17: 467
+    18: 467
     reference: Artikel 1 V. v. 28.09.2017 BGBl. I S. 3525.
   2019-01-01:
     6: 354
     12: 406
-    17: 476
+    18: 476
     reference: Artikel 1 V. v. 28.09.2017 BGBl. I S. 3525.
   2020-01-01:
     6: 369
     12: 424
-    17: 497
+    18: 497
     reference: Artikel 1 V. v. 12.09.2019 BGBl. I S. 1393.
   2021-01-01:
     6: 393
     12: 451
-    17: 528
+    18: 528
     reference: Artikel 1 V. v. 03.11.2020 BGBl. I S. 2344.
   2022-01-01:
     6: 396
     12: 455
-    17: 533
+    18: 533
     reference: Artikel 1 V. v. 30.11.2021 BGBl. I S. 5066.
   2023-01-01:
     6: 437
     12: 502
-    17: 588
+    18: 588
     reference: Artikel 1 V. v. 30.11.2022 BGBl. I S. 2130.
-unterhaltsvors_mindesteinkommen:
+abzugsrate_kindergeld:
   name:
-    de: >-
-      Monatliches Mindesteinkommen, um UHV für Kinder ab 13 Jahren zu erhalten
-    en: >-
-      Minimal income necessary to get child alimony advance for children 13 to 17 years.
-  description:
-    de: § 1 (1a) Nr. 2 Unterhaltsvorschussgesetz
-    en: null
-  unit: Euro
-  reference_period: Month
-  2017-01-01:
-    scalar: 600
-    reference: Art. 23 G. v. 14.08.2017 BGBl. I S. 3122.
-unterhaltsvors_altersgrenzen:
-  name:
-    de: Altersgrenzen für den Bezug von Unterhaltsvorschuss
-    en: age limits for receiving alimony payments
+    de: Verringerung des Kindesunterhalts durch Kindergeld
+    en: Reduction of child alimony payments due to child benefit
   description:
     de: >-
-      Kinder, die das 12. Lebensjahr noch nicht vollendet haben und bei einem
-      alleinerziehenden Elternteil leben haben Anspruch auf Unterhaltszahlungen. Zudem
-      können auch Kinder bis zur Vollendung des 18. Lebensjahres Unterhalt bekommen,
-      wenn das Elternteil mind. 600€ Einkommen hat.
+      Das Kindergeld zählt als Einkommen des Kindes, weshalb es bei der Berechnung von
+      Unterhalt Berücksichtigung findet. Erhält der betreuende Elternteil das
+      Kindergeld für ein minderjähriges Kind, so kann der andere unterhaltspflichtige
+      Elternteil das hälftige Kindergeld bei der Unterhaltsberechnung in Abzug bringen.
     en: >-
-      Children under the age of 12 living with a single parent are entitled to alimony
-      payments. In addition, children up to the age of 18 receive alimony payments if
-      the parent has an income of at least €600.
-  reference: § 1 Abs. 1, 1a UhVorschG
-  2017-01-01:
-    1: 6
-    2: 12
-    3: 18
+      Child benefit counts as the child's income, which is why it is taken into
+      account when calculating the child alimony payment. If the parent taking care of
+      the child receives the child benefit for a minor child, the other parent liable
+      for child alimony payments can deduct half of the child benefit when calculating
+      alimony.
+  reference: § 1612b BGB
+  unit: perecent
```

```suggestion
  unit: share
```
Percent would have to be 50 / 100, I'd hope.